spaceship/lib/spaceship/tunes/application.rb

```diff
@@ -236,6 +236,15 @@ def ensure_version!(version_number, platform: nil)
         end
       end
 
+      def reject_version_if_possible!
+        can_reject = edit_version.can_reject_version
+        if can_reject
```

Comment:
@joshdholtz What should be the right behaviour if we set `reject_if_waiting_for_review` as `true` but the app still can not be rejected because it's in 'can not reject` state? There is an intermediate state where the app is 'waiting for review' state but Apple doesn't let you reject the app.